kotlinx-coroutines-debug/src/DebugProbes.kt

```diff
@@ -24,16 +25,26 @@ import kotlin.coroutines.*
  * In practice, it means that for snapshotting operations in progress, for each concurrent coroutine either
  * the state prior to the operation or the state that was reached during the current operation is observed.
  *
- * ### Installed hooks
- * * `probeCoroutineResumed` is invoked on every [Continuation.resume].
- * * `probeCoroutineSuspended` is invoked on every continuation suspension.
- * * `probeCoroutineCreated` is invoked on every coroutine creation.
- *
  * ### Overhead
- *  * Every created coroutine is stored in a concurrent hash map and hash map is looked up and
+ *
+ *  - Every created coroutine is stored in a concurrent hash map and hash map is looked up and
  *    updated on each suspension and resumption.
- *  * If [DebugProbes.enableCreationStackTraces] is enabled, stack trace of the current thread is captured on
+ *  - If [DebugProbes.enableCreationStackTraces] is enabled, stack trace of the current thread is captured on
```

Comment:
While we mention' KDoc' from time to time, I'd like to hear your opinion here.

WDYT about making dashes the recommended (notably: KDoc spec, our libraries, IDEAs autoformatter and quickfixes) way for unordered lists instead of asterisks? I find it slightly more approachable than multiple asterisks (because of the one required by KDoc itself) especially when multiple levels of nesting are involved

Reply:
Sure: https://github.com/Kotlin/kotlinx.coroutines/pull/4027

Reply:
Filed https://youtrack.jetbrains.com/issue/KTIJ-28654/KDoc-autoformatting-replace-asterisks-with-dashes-for-list-declaration